app/views/physical_server/_main.html.haml

```diff
@@ -0,0 +1,7 @@
+= render :partial => "layouts/flash_msg"
+.row
+  .col-md-12.col-lg-6
+    = render :partial => "shared/summary/textual", :locals => {:title => _("Properties"), :items => textual_group_properties}
+    = render :partial => "shared/summary/textual", :locals => {:title => _("Relationships"), :items => textual_group_relationships}
+  .col-md-12.col-lg-6
+    TODO: other groups
```

Comment:
this file should not be needed at all. I have removed all of these when refactoring the textual summaries